apps/web/src/components/Feed/CuratedFeed.tsx

```diff
@@ -89,6 +89,7 @@ export default function CuratedFeed({ queryRef }: Props) {
       loadNextPage={loadNextPage}
       hasNext={hasPrevious}
       feedMode={'WORLDWIDE'}
+      showSuggestedProfiles={true}
```

Comment:
nit: can just pass in `showSuggestedProfiles` without setting to true

```js
  feedMode={'WORLDWIDE'}
  showSuggestedProfiles
  ...
```

Reply:
made the change!